src/app/visual/error/error-visual.component.html

```diff
@@ -50,7 +50,35 @@
       custom description
     </sky-error-description>
     <sky-error-action>
-      <button type="submit" class="sky-btn sky-btn-primary" >
+      <button
+        class="sky-btn sky-btn-primary"
+        type="submit"
+      >
+        custom action
+      </button>
+    </sky-error-action>
+  </sky-error>
+</div>
+
+<div class="error-screenshot" id="screenshot-error-custom-broken">
+  <sky-error
+    errorType="broken"
+  >
+    <sky-error-title
+      [replaceDefaultTitle]="true"
```

Comment:
Could we just call these `replaceDefault`? Since they are already on the `sky-error-title` and `sky-error-description` components?

Reply:
I believe Todd requested that they be the explicit versions

Reply:
Ok